auth/v1.js

```diff
@@ -315,11 +316,112 @@ api.declare({
     return res.status(404).json({message: "Client not found!"});
   }
 
-  // Reset accessToken
+  let added = _.without.apply(_, [input.scopes].concat(client.scopes));
+  if (!req.satisfies([added])) {
```

Comment:
Yeah, I'll loop back and do both API methods at once.  There's a skipped test that will go green, too.